tests/tests/test-pending-pool.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+
+import { createAndFinalizeBlock, customRequest, describeWithFrontier } from "./util";
+
+describeWithFrontier("Frontier RPC (Pending Pool)", `simple-specs.json`, (context) => {
```

Comment:
The import for us is `describeWithMoonbeam`

Reply:
Fixed (0e10887)